core/src/entities/cyclonedx/models/affects_inner.rs

```diff
@@ -0,0 +1,21 @@
+use serde::{Deserialize, Serialize};
+/*
+ * Generated by: https://openapi-generator.tech
+ */
```

Comment:
I think we should remove all these.

Reply:
The generated files or the comments?

Reply:
Nevermind, the comments but I have no idea why I focused on them so keenly.